packages/app/src/components/nominationBanner/nominationBanner.js

```diff
@@ -33,9 +33,7 @@ const NominationBanner = ({ nomination }) => {
               </span>
             </div>
             <div className="column name">
-              <button className=" decline-button" style={{ background: '#f72314', color: '#ffffff', border: '#929292', fontWeight: 'bold' }}>
-                Decline Application
-              </button>
+              <button className="decline-button">Decline Application</button>
```

Comment:
In the screenshot you provided it looks like the text is all uppercase. Not sure why that's the case given what you have here, but it should be first-letter-capitalized like the "Mark Stage as Complete" and "Close Application" buttons.

You can probably accomplish this by adding a `text-transform: none;` to the decline button's CSS.

Reply:
text-transform: none;  worked out perfectly! Updated the branch to reflect the changes you mentioned. Should be good to go